From: Alex Syrnikov <pioneer19@mailo.com>
Subject: Fix buildfile to build only build2 target
diff -ru build2-toolchain-0.14.0-a.0.20201106063545.a68b9bf6f066/build/bootstrap.build build2-toolchain-0.14.0-a.0.20201106063545.a68b9bf6f066.new/build/bootstrap.build
--- build2-toolchain-0.14.0-a.0.20201106063545.a68b9bf6f066/build/bootstrap.build       2020-11-04 12:26:41.000000000 +0300
+++ build2-toolchain-0.14.0-a.0.20201106063545.a68b9bf6f066.new/build/bootstrap.build   2020-11-06 21:57:37.788686680 +0300
@@ -3,8 +3,7 @@
 
 project = build2-toolchain
 amalgamation = # Disable amalgamation.
-subprojects  = {*/ -build/ -doc/ -tests/ -submodules/}
-subprojects += tests/*/
+subprojects = libbutl build2
 
 using version
 using config
diff -ru build2-toolchain-0.14.0-a.0.20201106063545.a68b9bf6f066/buildfile build2-toolchain-0.14.0-a.0.20201106063545.a68b9bf6f066.new/buildfile
--- build2-toolchain-0.14.0-a.0.20201106063545.a68b9bf6f066/buildfile   2020-11-04 18:31:10.000000000 +0300
+++ build2-toolchain-0.14.0-a.0.20201106063545.a68b9bf6f066.new/buildfile       2020-11-06 21:56:12.373265671 +0300
@@ -16,8 +16,7 @@
 #
 # See also subprojects in bootstrap.build.
 #
-d = libpkgconf/ libbutl/ build2/ libsqlite3/ libodb/ libodb-sqlite/ \
-libbpkg/ bpkg/ bdep/ doc/ libbuild2-*/
+d = libbutl/ build2/
 
 if ($build.meta_operation == 'dist')
   d += tests/*/
